plugins/outputs/cloudwatch/aggregator.go

```diff
@@ -144,7 +145,11 @@ func (durationAgg *durationAggregator) aggregating() {
 					m.distribution = distribution.NewDistribution()
 					err := m.distribution.AddEntryWithUnit(*m.Value, 1, *m.Unit)
 					if err != nil {
-						log.Printf("W! err %s, metric %s", err, *m.MetricName)
+						if errors.Is(err, distribution.ErrUnsupportedValue) {
+							log.Printf("W! err %s, metric %s", err, *m.MetricName)
+						} else if errors.Is(err, distribution.ErrUnsupportedWeight) {
+							log.Printf("D! err %s, metric %s", err, *m.MetricName)
+						}
```

Comment:
Super minor - if `UnsupportedWeight` is the only one we want at debug, then just make it `if`/`else` (instead of `else if`), just in case we add more errors later and forget to update this caller.

Reply:
Changed it so all non-value errors are logged as debug.